server/mergin/tests/utils.py

```diff
@@ -88,6 +88,7 @@ def create_project(name, workspace, user, **kwargs):
     pv = ProjectVersion(p, 0, user.id, changes, "127.0.0.1")
     db.session.add(pv)
     db.session.commit()
+    project_version_created.send(pv)
```

Comment:
Should not we trigger signal from ProjectVersion constructor instead?

Reply:
That would be probably the safest.

Reply:
Yes

Reply:
Indeed, I suggest some refactoring here, best to set it in one place and remove elsewhere.